src/config.ts

```diff
@@ -231,6 +231,20 @@ function loadCorsOptions(option: IProxyOption): CorsOptions {
     return computedCorsOptions;
 }
 
+function loadHttpOptions(option: IProxyOption): HttpOptions {
+    if (option.httpOptions) {
+        return option.httpOptions;
+    }
+
+    if (process.env.HTTP_OPTIONS_REJECT_UNAUTHORIZED) {
+        return {
+            rejectUnauthorized: safeBoolean(process.env.HTTP_OPTIONS_REJECT_UNAUTHORIZED, false)
+        };
```

Comment:
This can return `{ httpOptions: {...} }` or `{}` in order to simplify logic in L:344
(to be `...loadHttpOptions(option),`)

Reply:
Done this change